assets/js/src/Popover.js

```diff
@@ -13,6 +13,10 @@ Kanboard.Popover.prototype.listen = function() {
         self.close(e);
     });
 
+    $(document).on("click", "#popover-container", function(e) {
+        self.close(e);
```

Comment:
Can you remove this part for now? I prefer to put back this feature only if we introduce an alert box for modal box that contains a form.

Thanks